### ISSUE TITLE
button: shows black background on focus

### ISSUE THREAD

**Issue by KaidenR** [pluralsight/design-system#290]
### Expected behavior

The background should remain the same when focused.

### Actual behavior

When a flat button on a colored background is focused the background of the button changes to black because the focus ring adds the background.

### Steps to repro

1. Add a flat button to a container with a background
2. Focus the button - the button's background will go black

![image](https://user-images.githubusercontent.com/679346/47105010-0cb7b880-d200-11e8-8167-e9efc7509afc.png)
![image](https://user-images.githubusercontent.com/679346/47105021-13dec680-d200-11e8-8a81-5172cfb90a23.png)
